src/test/scala/chiselTests/StrongEnum.scala

```diff
@@ -376,6 +376,7 @@ class StrongEnumAnnotationSpec extends FreeSpec with Matchers {
   import chisel3.experimental.EnumAnnotations._
   import firrtl.annotations.ComponentName
 
+  /*
   "Test that strong enums annotate themselves appropriately" in {
```

Comment:
[nit] ScalaTests's `ignore` is super useful for this type of thing and generally better than commenting out tests:
```scala
ignore should "Test that strong enums annotate themselves appropriately" in {
// ...
}
```

That way, the test is just skipped, but still shows up when you run tests and counts in the "ignored" bucket. Some more info from [ScalaTest](http://www.scalatest.org/user_guide/tagging_your_tests).